lib/mp4/transmuxer.js

```diff
@@ -503,9 +503,9 @@ VideoSegmentStream = function(track) {
 
     // For the last frame, use the duration of the previous frame if we
     // have nothing better to go on
-    if (frames.length &&
+    if (frames.length && (
```

Comment:
Could you move this paren to the next line so that this can match the rest of the code in our plugins.